frontend/src/pages/project/main/ProjectItem.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { ReactElement } from "react";
+import classes from "./ProjectItem.module.scss";
+import { ProjectItemBadge } from "./ProjectItemBadge";
+
+type ProjectItemProps = {
+  content: { title: ReactElement | string; timestamp: ReactElement | string };
```

Comment:
This component is great! One tiny suggestion - I would prefer if `title` and `timestamp` were defined at a top level, like so:
```ts
type ProjectItemProps = {
  title: ReactElement | string;
  timestamp: ReactElement | string;
}
```

Also I think that `timestmap` won't be used only for the timestamp (date), but could also be used for some generic text (e.g. sub-title). That's why I think it's better to rename that prop to something like `subTitle`.